app/src/main/java/protect/card_locker/DBHelper.java

```diff
@@ -47,6 +47,7 @@ public static class LoyaltyCardDbIds {
         public static final String LAST_USED = "lastused";
         public static final String ZOOM_LEVEL = "zoomlevel";
         public static final String ARCHIVE_STATUS = "archive";
+        public static final String USAGE_NUMBER = "usagenumber";
```

Comment:
I think I'd much prefer this to be named `USAGE_COUNT` and `usagecount`. To me that is more logical.

Reply:
Done :)